build.gradle

```diff
@@ -16,4 +16,32 @@ allprojects {
             sourceCompatibility = JavaVersion.VERSION_11
         }
     }
+    pluginManager.withPlugin('java-library') {
+        configurations {
+            // See https://github.com/gradle/gradle/issues/10861
+            // To avoid publishing the internal-platform
+            internalPlatform {
```

Comment:
now you are just copy/pasting gradle snippets from other projects ? :joy:

Why are you defining `internalPlatform` here, when it is not used anywhere in this project ?